commands/list.go

```diff
@@ -93,7 +97,24 @@ type listResult struct {
 func (res *listResult) Result() {}
 
 func (res *listResult) String() string {
-	return RenderTemplate(listResultT, res)
+	template := RenderTemplate(listResultT, res)
+	formattedTemplate, err := convertTabs(template)
+	if err != nil {
+		logrus.Errorf("%s", err)
```

Comment:
```suggestion
		logrus.Panicf("Failed to render response: %s", err)
```

Reply:
@BupycHuk @rnovikovP I couldn't figure out the reason for the build failure. Seems like you have some static analyzers which check for code smell. Is it possible to make those static analyzers were able to leave comments on Github for code smells?

Reply:
It does, but because your PR is made from different repository it's failing. We are working on fixing it.
For now you can run it locally by calling `make check-all`